src/types.ts

```diff
@@ -1,44 +1,39 @@
-export type VoiceStartMessage = {
-  type: 'voiceStart';
+export type MeasureStartMessage = {
+  msgType: 'measureStart';
+  width: string;
+  staves: string;
```

Comment:
The `<staves>` element has a `nonNegativeInteger` for its data type. https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/staves/

Would you change this type to a `number` please?

Reply:
done